From 2bbb660669791be3c1a884c1a730be8f5f58caec Mon Sep 17 00:00:00 2001
From: Mario Sanchez Prada <mario@endlessm.com>
Date: Thu, 28 Jan 2016 13:53:08 +0000
Subject: [PATCH] memory: Enable the tab discards feature

This allows manually discarding tabs from chrome://discards as well
as automatic tab discards once a certain level of "memory pressure"
is reached.
---
 chrome/browser/resource_coordinator/tab_manager.cc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/chrome/browser/resource_coordinator/tab_manager.cc b/chrome/browser/resource_coordinator/tab_manager.cc
index 6f76e5aad4fb8..e08dfbc98773a 100644
--- a/chrome/browser/resource_coordinator/tab_manager.cc
+++ b/chrome/browser/resource_coordinator/tab_manager.cc
@@ -171,7 +171,8 @@ void TabManager::Start() {
 
 // MemoryPressureMonitor is not implemented on Linux so far and tabs are never
 // discarded.
-#if defined(OS_WIN) || defined(OS_MAC) || defined(OS_CHROMEOS)
+#if defined(OS_WIN) || defined(OS_MAC) || defined(OS_CHROMEOS) || \
+    defined(OS_LINUX)
   // Don't handle memory pressure events here if this is done by
   // PerformanceManager.
   if (!base::FeatureList::IsEnabled(
-- 
2.32.0

